.github/workflows/sanitize.yml

```diff
@@ -30,12 +30,13 @@ jobs:
       - name: Build
         run: |
           # build tests , dont run them
-          $GITHUB_WORKSPACE/tools/iceoryx_build_test.sh build-strict build-all sanitize clang
+          sudo $GITHUB_WORKSPACE/tools/iceoryx_build_test.sh build-strict build-all sanitize clang test-add-user
```

Comment:
When we build the sanity check do we also build all examples to ensure that they do not have a compile problem in sanitize mode?

Reply:
all examples are build with the sanitizer flags